service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -54,25 +49,35 @@
     @Autowired
     private StorageConnector storageConnector;
 
-    @Test
-    public void When_NewAccessDocumentTaskIsStarted_Expect_DatabaseUpdatedAndAddedToObjectStore() throws IOException {
+    private EhrExtractStatus setupDatabase() {
         var ehrExtractStatus = EhrExtractStatusTestUtils.prepareEhrExtractStatus();
-        ehrExtractStatusRepository.save(ehrExtractStatus);
+        ehrExtractStatus.setGpcAccessDocument(EhrExtractStatus.GpcAccessDocument.builder()
+            .documents(List.of(
```

Comment:
Same as above.

Reply:
fine for using builder to create test fixture